scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -121,8 +121,8 @@
             <th>Reduced penetrance</th>
             <th>Mosaicism</th>
             <th>Entry version</th>
-            <th>Standard inheritance models</th>
-            <th>Custom inheritance models</th>
+            <th>Manual inheritance (standard terms)</th>
+            <th>Manual inheritance (custom terms)</th>
```

Comment:
Manual inheritance (free text terms) - maybe enven a tooltip contrasting to OMIM and HPO?

Reply:
I renamed the col to `Manual inheritance (free text terms)`. About the tooltip, I wouldn't spend too much time on it since we are going to remove the column anyway..